src/WordPress/DataStore.php

```diff
@@ -18,6 +20,13 @@ class DataStore implements DataStoreInterface
     public function __construct(DefaultLogger $logger)
     {
         $this->logger = $logger;
+
+        // Create temp object to be able to call createPluginSettingObject
+        $config = new DefaultConfig('[]');
+        $wordpressAPI = new WordPressAPI($this);
+        $wordpressIntegration = new DefaultIntegration($config, $wordpressAPI, $this, $logger);
+
+        $this->tempPlugin = new Plugin($wordpressIntegration);
```

Comment:
Every solution I came up with sucked which included using the current version of cloudflare-plugin-backend. For fast fix I chose this one.